### DISCUSSION TITLE
Suggestion - add dates to tags list dropdown

### DISCUSSION THREAD

**Discussion by yairEO** [community/community#12776] [Other Feature Feedback, Questions, & Ideas]
When viewing the *list of tags*, in the *branches/tags* button dropdown, it is highly beneficial to show the "last changed" **date** next to every tag (either on hover only or not, doesn't matter). 

There's plenty of space there in the dropdown, to the right of the printed version number, for a small date to fit, and even if not, at least  a title tooltip when hovering a list item, which would show the date, would be very nice addition :) 

Of course this data is available in the dedicated Tags page, but why bother going there if it can be already viewed comfortably in the dropdown without leaving the current context of the page. Many times all that's needed - a quick peak at some versions' dates.

![image](https://user-images.githubusercontent.com/845031/158054079-3bfcc563-fb54-484b-b5f7-f9462f5e13dd.png)

**Comment by lirantal** [comment]
That's a good idea!

**Comment by rafmim** [comment]
A very useful quick win imho :)
